Server/Plugins/APIDump/APIDesc.lua

```diff
@@ -14877,6 +14877,40 @@ end
 					},
 					Notes = "Returns the {{Globals#BlockFaces|eBlockFace}} that corresponds to the given {{Globals#BlockFaces|eBlockFace}} after mirroring it around the Y axis (or rotating 180 degrees around it).",
 				},
+				MirrorBlockFaceXY =
+				{
+					Params =
+					{
+						{
+							Name = "eBlockFace",
+							Type = "eBlockFace",
+						},
+					},
+					Returns =
+					{
+						{
+							Type = "eBlockFace",
+						},
+					},
+					Notes = "Returns the {{Globals#BlockFaces|eBlockFace}} that corresponds to the given {{Globals#BlockFaces|eBlockFace}} after mirroring it around the XY plane (or rotating 180 degrees around it).",
```

Comment:
So is it mirroring or rotating? In the general sense, those two operations are different, though for Minecraft they may be the same?

Reply:
Indeed, this comment is wrong.

For most blocks, this applies. But if you want to rotate blocks with the ability to be asymmetric you don't have 180 degree rotation but mirroring. For examples vines.

Example viewed from the top:
`_|` should mirror to `|_` but rotation does yield `|^`. Imagine the `^` is a horizontal line at the top.